### ISSUE TITLE
Last page never loads when zoom level is below ~0.4

### ISSUE THREAD

**Issue by wkarras** [VadimDez/ng2-pdf-viewer#261]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [X] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

It's inconsistent at what zoom level this occurs, but when at about 0.5 or lower, the last page of a pdf never loads - you just get the loading spinner indefinitely

I was able to reproduce this on the demo page https://vadimdez.github.io/ng2-pdf-viewer/ by lowering the zoom level to 0.4 and the last page of the test pdf doesn't load

**Comment by Celid** [comment]
I can confirm that the size has an impact : I have set a max-height to .pdfViewer, and the last pages never load (only the two first are readable)

**Comment by VadimDez** [comment]
Fixed in `6.0.0` See https://github.com/VadimDez/ng2-pdf-viewer/blob/master/CHANGELOG.md#600
